test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -677,19 +685,33 @@ fi
 
 if [ "$patch_version" = true ]; then
     for b in ${patch_version_upgrade_downgrade_release_branches[@]}; do
-        # Build the tip of the release branch and run test to genearte data files
-        (build_branch $b)
+        # Build the tip of the release branch and run test to generate data files
+        (build_branch $b 1)
         (run_test_checkpoint "$b" "row")
 
         # Pick a patch version from the list of patch versions for the release branch
-        cd $b; pv=$(pick_a_version $b); cd ..
-        (build_branch $pv)
+        cd $b; pv=$(pick_a_version $b); echo "$pv"; cd ..
```

Comment:
If you'd like to dump out the picked version, how about extending the output to e.g. "The picked version: $pv"?

Reply:
Sure, added that!